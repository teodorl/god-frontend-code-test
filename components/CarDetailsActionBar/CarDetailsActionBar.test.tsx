import React from 'react';
import { render } from '../../test-utils';
import { CarDetailsActionBar } from './';

const mockProps = {
    id: 'xc-90',
};

describe('<CarDetailsActionBar /> spec', () => {
    it('renders the component', () => {
        const container = render(<CarDetailsActionBar {...mockProps} />);
        expect(container.firstChild).toMatchSnapshot()
    });
});
