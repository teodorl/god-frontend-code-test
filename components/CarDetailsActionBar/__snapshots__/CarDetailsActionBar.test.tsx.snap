// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CarDetailsActionBar /> spec renders the component 1`] = `undefined`;
